test/run_pass/test_process_send.js

```diff
@@ -0,0 +1,17 @@
ebase:device", "offline": false, "driverId": "ssdp", "driverAccountId": "test_account_test_account", "actions": { "switch": ["on", "off"], "colo": ["num"], "volume": ["up", "down", "max", "min", "num"] }, "state": { "switch": null, "color": null }, "alias": "电灯1", "__v": 0, "removeAt": null, "sortId": "5b557377e0050c005991b572", "enTags": [], "zhTags": ["电灯1", "灯"], "tags": [], "disabled": false, "states": [], "capabilities": [], "id": "rJblt7xm4X", "room": { "id": null }, "icon": "https://s.rokidcdn.com/homebase/upload/B1zxcstbtxz.png" }, { "_id": "HJxxY7g7NX", "updated_at": "2018-07-23T07:11:51.380Z", "created_at": "2018-07-23T06:19:35.917Z", "homeId": "test_home_test_home", "rokidId": "test_rokid_test_rokid", "userId": "testid_testid_testid_testid_testid_testid", "name": "电灯0", "type": "light", "deviceId": "ssdp-test-device-0", "deviceInfo": { "bridgeId": "SSDP-test-bridge::homebase:device" }, "parent": "SSDP-test-bridge::homebase:device", "offline": false, "driverId": "ssdp", "driverAccountId": "test_account_test_account", "actions": { "switch": ["on", "off"], "colo": ["num"], "volume": ["up", "down", "max", "min", "num"] }, "state": { "switch": null, "color": null }, "alias": "电灯0", "__v": 0, "removeAt": null, "sortId": "5b557377e0050c005991b571", "enTags": [], "zhTags": ["电灯0", "灯"], "tags": [], "disabled": false, "states": [], "capabilities": [], "id": "HJxxY7g7NX", "room": { "id": null }, "icon": "https://s.rokidcdn.com/homebase/upload/B1zxcstbtxz.png" }, { "_id": "test_rokid_test_rokid", "updated_at": "2018-07-23T11:01:13.238Z", "created_at": "2018-07-18T11:40:18.421Z", "homeId": "test_home_test_home", "userId": "testid_testid_testid_testid_testid_testid", "deviceId": "0602041822000014", "deviceInfo": { "wifi": "", "ip": "10.89.2.78", "standAlone": true, "deviceTypeId": "060F941561F24278B8ED71733D7B9507", "platform": "linux" }, "offline": false, "type": "rokid", "__v": 0, "name": "若琪-014", "removeAt": null, "sortId": "5b4f27222fcf42006174897b", "enTags": [], "zhTags": [], "tags": [], "disabled": false, "states": [], "capabilities": [], "id": "test_rokid_test_rokid", "alias": null, "room": { "id": null }, "icon": "https://s.rokidcdn.com/homebase/upload/rJ9cjFWteM.png" }, { "_id": "HkeplDOV77", "updated_at": "2018-07-16T05:39:02.344Z", "created_at": "2018-07-12T06:36:36.907Z", "homeId": "test_home_test_home", "userId": "testid_testid_testid_testid_testid_testid", "deviceId": "0401051822000053", "deviceInfo": { "wifi": "ROKID.TC", "ip": "10.89.7.187", "standAlone": true, "deviceTypeId": "AA12D7458EAD4AC4A8BF8ABB96FD8B74", "platform": "linux" }, "offline": true, "type": "rokid", "__v": 0, "name": "若琪-053", "removeAt": null, "sortId": "5b46f6f41c343d005a8e743d", "enTags": [], "zhTags": [], "tags": [], "disabled": false, "states": [], "capabilities": [], "id": "HkeplDOV77", "message": { "text": "若琪离线", "style": "normal" }, "alias": null, "room": { "id": null }, "icon": "https://s.rokidcdn.com/homebase/upload/SkYcjKbtxf.png" }], "isEnd": true }
```

Comment:
This causes the file is too large, move to another JSON file, and `require` here would be readable :)